docker/prowbazel/README.md

```diff
@@ -56,3 +56,4 @@ Our dependency on this script is because it appropariately writes test job resul
 * 0.5.10: Install cmake in /usr/local/bin
 * 0.5.11: Update clang and libc++ to 9.0 and preload Bazel 1.0.0
 * 0.5.12: Install clang and libc++ from the official releases instead of packages.
+* 0.5.14: Install goimports and golangci-lint
```

Comment:
I'd recommend using bazel to install and manage goimports and golangci-lint for a bazel image

Reply:
Ah, thanks for the pointer! I think it makes more sense to use the bazel rules for istio/proxy. When proxy uses an image built from `build-tools`, we can switch to go-lint target from common files. 

But anyway, let's merge this to make everything here in sync with images in gcr, there is no harm to do so.